api/HTMLInputElement.json

```diff
@@ -1468,53 +1468,6 @@
           }
         }
       },
-      "onsearch": {
-        "__compat": {
-          "support": {
-            "chrome": {
-              "version_added": "1"
-            },
-            "chrome_android": {
-              "version_added": "18"
-            },
-            "edge": {
-              "version_added": "79"
-            },
-            "firefox": {
-              "version_added": false
-            },
-            "firefox_android": {
-              "version_added": false
-            },
-            "ie": {
-              "version_added": false
-            },
-            "opera": {
-              "version_added": "15"
-            },
-            "opera_android": {
-              "version_added": "14"
-            },
-            "safari": {
-              "version_added": "1.3"
```

Comment:
This doesn't match the version in the "search_event" entry that remains. Can you check which is correct?

Reply:
Ack, this one slipped through -- the "1.3" came from tests I did a while back, and the "2" came from https://github.com/mdn/browser-compat-data/pull/11755#issuecomment-905677809.  I'm going to stick with "2" on this one for the same reasons.

Reply:
Alright, let's stick with Safari 2!